pypi-dependencies.json

```diff
@@ -2,223 +2,5 @@
     "name": "pypi-dependencies",
     "buildsystem": "simple",
     "build-commands": [],
-    "modules": [
```

Comment:
Mmm, this seems pretty wrong: you literally removed all the python dependencies. I would be really surprised if this would run correctly.

Reply:
Yes, just noticed that. I'll see what I did wrong when running `refresh-pip-dependencies.sh`

Reply:
You need to run it from the main project (this is just a submodule): 
https://gitlab.com/leinardi/gwe

Reply:
The script need access to this: https://gitlab.com/leinardi/gwe/-/blob/release/requirements.txt

Reply:
`"sources": []` doesn't seem correct :thinking:

Reply:
yep, and `pygobject==3.34.0"` is now out of sync. My advice is to update first the requirements.txt on gitlab, check if everything still works, and then update the flatpak manifest...

Reply:
Thanks. I will try.